File: others/ruby/practice/fizzbuzz.rb
FB = [*(1..10)]
B = [*(1..50)]
F = [*(1..50)]
for i in 1..100 do
	if i / 15 == FB[i/15] then
		print "Fizzbuzz"
	elsif i / 5 == B then
		print "Buzz"
	elsif i / 3 == F then 
		print "Fizz"
	else
		print i
	end
	print " "
end

File: forth/fizzbuzz.fs
\ #! /usr/bin/env gforthrun

: fizzbuzz
  1 Do
    i 5 mod 0=
    i 3 mod 0=
    2dup or 0=
    if i 1 .r then
    if ." Fizz" then
    if ." Buzz" then
  44 EMIT
  LOOP ;

101 fizzbuzz
